driver/driver.feature

```
@@ -335,7 +334,7 @@ Feature: TypeDB Driver
       """
     When wait 4 seconds
     Then transaction is open: false
-    Then typeql read query; fails with a message containing: "no open transaction"
```

The messages are different in two endpoints, and I don't want to merge them. It's hard to write this tests wrongly, and there is an additional check above, so it's fine to ignore the error mesage.